assets/sass/alaveteli-org.scss

```diff
@@ -892,3 +894,91 @@ h6:hover .header-link {
     margin-right: 4px;
   }
 }
+
+/* LANGUAGE SWITCHER */
+/* ui-dropdown-list */
+.ui-dropdown-list {
```

Comment:
This section could be refactored to use SCSS nesting, it would make it a bit easier to read